constraints.txt

```diff
@@ -1,4 +1,4 @@
-ansible==2.3.2.0
+ansible==2.5.2
```

Comment:
Why the upgrade of Ansible?

Reply:
This impetus was custom jinja tag strings, which aren't available in 2.3.